### DISCUSSION TITLE
What cron jobs if any do I need to setup for OpenMage v19?

### DISCUSSION THREAD

**Discussion by djixas** [OpenMage/magento-lts#3193] [Questions & Answers] [answered]
I couldn't find any info.

**Comment by addison74** [comment, marked as answer]
You must read the Magento 1 user manual. If you use DDEV @sreichel has already created a README file which contains a tutorial for cron jobs. If you use Linux then you must add a line for a user in the cron table. For example 

```crontab -u <user_name> -e```

Once the editor opens add this line

```php
*/3 * * * * /bin/sh /home/<user_name>/public_html/cron.sh

```

Restart cron service

```systemctl restart cron.service```
